### ISSUE TITLE
\RequirePackage[tagged]{accessibility_meta} prevents document from compiling

### ISSUE THREAD

**Issue by aronadaal** [AndyClifton/AccessibleMetaClass#2]
Hi,
I just tried to compile the demo file with an up-to-date MikTex and got the following output:
http://pastebin.com/kKEnfiVD
Around line 1197 I got the error "! LaTeX Error: Missing \begin{document}."

Bye, Holger

**Comment by aronadaal** [comment]
the same on a up-to-date MacTeX...
http://pastebin.com/SKwdDtPa

**Comment by AndyClifton** [comment]
Confirmed on my installation too (TexLive 2014). Currently not clear what's causing this.

**Comment by AndyClifton** [comment]
The bug is introduced in the _accessibility_meta.sty_ file. The compiler problem disappears when I comment the _\RequirePackage[tagged]{accessibility_meta}_ (line 443) line out in _meta.cls_. That's unfortunate, as the _accessibility_meta.sty_ file is kind of the point of this class demo!

**Comment by scohen345** [comment]
Hi Andy,

Is there any update on the problem above using accessibility-meta? I also get the "Missing \begin{document}" error when I run pdfLaTeX on DemoAccMetaClass.tex without making any changes to any files. I am using MikTeX 2.9 from within TeXworks.

When I compile ArticleTest.tex, I get the following error.

! Font \csname\endcsname=dummy-space at 1.00375pt not loadable: Metric (TFM) fi
le not found.
\@EveryShipout@Output ...@Org@Shipout \box \@cclv                                      
l.181 \begin{document}

When I insert 

\usepackage[tagged]{accessibility-meta}

just before the begin{document} in some of own LaTeX files, I get the following error.

Package accessibility Warning: Die gew?hlte Sprache (english) wird vom Adobe Ac
robat Reader 6.0 nicht unterst?tzt. on input line 112.
! Package accessibility Error: Changed paragraphs, rerun to get it right.

Re-running does not fix the problem.

Any suggestions?

Thanks.
-Scott

P.S. The first error that I got on DemoAccMetaClass.tex was 

! LaTeX Error: File `accessibilityMeta.sty' not found.

I noticed that the file in the directory is actually called accessibility-meta.sty. I made a copy of the file, named the copy accessibilityMeta.sty, and tried pdfLaTeX again. That's when I get the Missing Document error.

**Comment by neingeist** [comment]
@scohen345 

There are multiple different errors:

1. `Font \csname\endcsname=dummy-space at 1.00375pt not loadable: Metric (TFM) fi
le not found.`: look for dummy-space.tfm on your system. Your TeX installation might not be new enough.

2. `Die gew?hlte Sprache (english) wird vom Adobe Acrobat Reader 6.0 nicht unterst?tzt.`: That is a misleading error in German. I will look into it.

3. `Error: Changed paragraphs, rerun to get it right.`: This should be a warning, I opened PR #5 to downgrade it to a warning.

4. `Error: File `accessibilityMeta.sty' not found.`: Should not happen, maybe happened with an earlier checkout of the repo?

**Comment by neingeist** [comment]
@scohen345 
`Die gew?hlte Sprache (english) wird vom Adobe Acrobat Reader 6.0 nicht unterst?tzt.` – I've submited PR #6 for a more helpful message. The fix on your end is to use the babel package.

**Comment by neingeist** [comment]
@AndyClifton I believe this issue can be closed now?

**Comment by AndyClifton** [comment]
@neingeist Thanks! Please feel free to raise and address other issues!
